tests/integration/__init__.py

```diff
@@ -201,6 +201,9 @@ def _get_dse_version_from_cass(cass_version):
     CASSANDRA_VERSION = Version(mcv_string) if mcv_string else cassandra_version
     CCM_VERSION = mcv_string if mcv_string else cv_string
 
+    CASSANDRA_VERSION = Version(mcv_string) if mcv_string else cassandra_version
+    CCM_VERSION = mcv_string if mcv_string else cv_string
+
```

Comment:
Those seem to be duplicated from the lines above

Reply:
fixed